scripts/build-retention/build_retention.groovy

```diff
@@ -0,0 +1,52 @@
+pipeline {
```

Comment:
Please move this file to jenkins/automation/build-retention

Reply:
Done